index.html

```diff
@@ -0,0 +1,172 @@
+            <p>
+             Lorem ipsum dolor sit amet consectetur adipisicing elit. Delectus animi eos beatae nemo quasi ullam provident maiores, corporis iste perferendis laudantium commodi corrupti omnis consequatur inventore impedit incidunt aut similique.
+            </p>
+            <code>
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Fugit similique sapiente, totam mollitia culpa assumenda consequatur! Eaque alias ullam, tempore dolor cum quasi atque error a, quo autem dignissimos sint?
+
+            </code>
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Possimus pariatur assumenda odit recusandae perferendis rem magnam nesciunt at est aperiam aspernatur non aliquid facilis perspiciatis dolores facere molestias, voluptate quibusdam.</p>
+            <ul>
+                <li>list 1</li>
+                <li>list 2</li>
+                <li>list 3</li>
+                <li>list 4</li>
+                <li>list 5</li>
+            </ul>
+        </section>
+        <section class="main-section" id="Introduction">
+
+            <header>Introduction</header>
+            <p>
+                Lorem ipsum, dolor sit amet consectetur adipisicing elit. Tempore aut facere itaque eveniet libero quod sint nesciunt architecto. Distinctio dolores sint quibusdam vel quaerat debitis expedita excepturi adipisci dolorum maxime?
+            </p>
+            <code>
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Fugit similique sapiente, totam mollitia culpa assumenda consequatur! Eaque alias ullam, tempore dolor cum quasi atque error a, quo autem dignissimos sint?
+                
+            </code>
+            <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Quisquam quia dicta dignissimos eaque mollitia sapiente est! Aliquid praesentium, aut veritatis molestiae modi fugiat. Deserunt eos recusandae veritatis amet inventore qui.</p>
+        </section>
+        <section class="main-section" id="JavaScript_and_Java">
+            <header>JavaScript and Java</header>
+            <p> Lorem ipsum dolor sit amet consectetur adipisicing elit. Consectetur vitae natus quam ab dolore laborum vel accusamus aliquam architecto deserunt hic voluptates sunt, ut perferendis quae, quia earum quas exercitationem.</p>
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Possimus sunt quisquam libero suscipit minus, magnam esse odit neque dolorem ipsam aliquam soluta? Nisi necessitatibus deserunt non illum! Corrupti, nesciunt optio.
+
+            </p>
+            <code>
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Fugit similique sapiente, totam mollitia culpa assumenda consequatur! Eaque alias ullam, tempore dolor cum quasi atque error a, quo autem dignissimos sint?
+                
+            </code>
+        </section>
+        <section class="main-section" id="comments">
+            <header>comments</header>
```

Comment:
### Header
Try to use the semantic tags in the correct form. <header> is mainly used on the top of the page, normally as a navigation bar or "mylogo" and menu section.